tests/lua-tests/src/SpritePolygonTest/SpritePolygonTest.lua

```diff
@@ -690,7 +690,7 @@ end
 
 function SpritePolygonPerformanceTestDynamic:initIncrementStats()
     self._incVert = self._info:getVertCount()
-    self._incTri  = self._info:getTriaglesCount()
```

Comment:
@zilongshanren I suppose it is not necessary to do. It has already been updated automatically as replacement:
https://github.com/cocos2d/cocos2d-x/commit/b340e48cb14d1beda756f4455664bcbc8d53d1c2?diff=unified#diff-86661c5b154867b8b3ece03f37536f87R23